### ISSUE TITLE
update version in npm

### ISSUE THREAD

**Issue by davepacheco** [joyent/manta-thoth#131]
Users have been confused by the fact that there's a thoth package in npm ("manta-thoth"), but it's ancient.  We should probably either remove that or update it.

**Comment by trentm** [comment]
@bcantrill If you could give some of the rest of us publish rights then we could handle this for you.

```
npm owner add dap manta-thoth
npm owner add trentm manta-thoth
npm owner add jclulow manta-thoth
```

**Comment by jclulow** [comment]
When we do this, we should release `1.0.0`.

**Comment by jclulow** [comment]
@bcantrill ping?

**Comment by trentm** [comment]
We have published rights now, and #144 (CR https://cr.joyent.us/#/c/942) will update and publish 1.0.0.
